webapp/store/system/i18n/reducer.js

```diff
@@ -0,0 +1,9 @@
+import { exportReducer } from '@webapp/utils/reduxUtils'
+
+import { appPropsChange } from '@webapp/app/actions'
+
+const actionHandlers = {
+  [appPropsChange]: (state, { i18n }) => i18n || state,
```

Comment:
here we need its own action. is it possible?

Reply:
yes I felt that.. because appPropsChange is too general we need to replace this appPropsChange for a parallel dispatch, not sure if I have to it now

Reply:
just leave it as it is now. no worries